backend/test_auth.py

```
@@ -0,0 +1,55 @@
+BASE_URL = "http://localhost:8000/api/v1"
+
+def test_signup():
+    """Test user signup."""
+    payload = {
+        "email": "test@example.com",
+        "password": "testpassword123",
+        "full_name": "Test User"
+    }
+
+    try:
+        response = requests.post(f"{BASE_URL}/auth/signup", json=payload)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)